src/live_executor.cc

```diff
@@ -8,20 +8,262 @@
 #include "out_of_order_engine.h"
 #include "receive_arbiter.h"
 #include "system_info.h"
+#include "tracy.h"
 #include "types.h"
 #include "utils.h"
 
-#include <memory>
+#include <deque>
 #include <optional>
 #include <string>
 #include <unordered_map>
 #include <vector>
 
 #include <matchbox.hh>
 
+
 namespace celerity::detail::live_executor_detail {
 
+#if CELERITY_TRACY_SUPPORT
```

Comment:
Could this be moved to a separate file for readability purposes (i.e. to, as much as possible, keep the core functionality of the live executor separate from concerns related to Celerity integration).

Reply:
I had considered this in the beginning, but since I ended up with quite a lot of Tracy-conditional code within executor member functions that reference these structures directly anyway, I decided against it.